tests/camb/test_wrapper.py

```diff
@@ -80,6 +81,14 @@ def test_hubble_distance(cosmo, compare):
     )
 
 
+def test_critical_density0(cosmo, compare):
+    np.testing.assert_allclose(
+        cosmo.critical_density0,
+        compare.critical_density0.to(u.solMass / u.Mpc**3).value,
```

Comment:
No need for the units import
```suggestion
        compare.critical_density0.to("Msun Mpc-3").value,
```

Reply:
TIL